pandas/tests/extension/base/dim2.py

```diff
@@ -191,7 +192,11 @@ def test_reductions_2d_axis0(self, data, method):
             kwargs["ddof"] = 0
 
         try:
-            result = getattr(arr2d, method)(axis=0, **kwargs)
+            if method == "mean":
```

Comment:
We have 2 all NA rows in our array that causes numpy to raise a RuntimeWarning with axis=0